draft-ietf-scone-protocol.md

```diff
@@ -365,17 +372,13 @@ if they have exceeded limits.
 
 The choice of 67 seconds, as a prime number,
 also ensures that problems in tuning applications
-are not synchronized with other periodic effects.
+are not synchronized with other periodic effects
+that might be measured in whole seconds.
+This includes segment length or key frame intervals in video applications,
+but also includes timers for middleboxes; see {{Section 4.3 of ?RFC4787}}.
 Any repeating phenomenon at this interval is therefore
 unlikely to be due to other periodic effects.
```

Comment:
The sentence in the middle is much improved, but I don't fancy much the undefined "this" in the last sentence. Matter of taste, but I would write "at a 67 seconds interval".

And just to be facetious, why not 61, or 71, or 73?

Reply:
There's no real reason for it to be 67 over any of those.  Also, 68 probably wouldn't be so bad either.